swiftui/ChartboostMediationDemo/Ads/Banner/BannerAdView.swift

```diff
@@ -86,10 +86,14 @@ struct BannerAdView: View {
                 isBusy = true
             case .failed(let message, let error):
                 isBusy = false
-                failureMessage = "\(message): \(error.localizedDescription)"
+                failureMessage = "\(message): \(error?.localizedDescription ?? "")"
```

Comment:
```suggestion
if let error {
                failureMessage = "\(message): \(error.localizedDescription)"
} else {
                failureMessage = message
}
```

You might also be able to handle this with pattern matching in the switch statement.